src/test/regress/expected/failure_on_create_subscription.out

```diff
@@ -73,12 +73,15 @@ SELECT count(*) FROM t;
 (1 row)
 
 -- Verify that shard can be moved after a temporary failure
+-- cleanup leftovers
 SELECT citus.mitmproxy('conn.allow()');
  mitmproxy
 ---------------------------------------------------------------------
 
 (1 row)
 
+CALL citus_cleanup_orphaned_resources();
+NOTICE:  cleaned up 4 orphaned resources
```

Comment:
Is this call still needed?

Reply:
Missed that, deleted, thanks!
Deleted other needless calls as well